Bottom: 6a9446a4008c8d1554affab6f0e38c321bbc0f85
Top:    77042c85570cd022e3acfb8c96bfe4a2c6472f8a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 19:14:36 +0000

Refresh of fix-some-warnings-due-to

---

diff --git a/src/misc.c b/src/misc.c
index 0e46b98..9230b16 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -614,7 +614,7 @@ SetTextBoundingBox (FontType *FontPtr, TextType *Text)
 {
   SymbolType *symbol = FontPtr->Symbol;
   unsigned char *s = (unsigned char *) Text->TextString;
-  int i;
+  Cardinal i;
   int space;
 
   Coord minx, miny, maxx, maxy, tx;
